guacamole-docker/bin/start.sh

```diff
@@ -707,6 +722,11 @@ END
     exit 1;
 fi
 
+# Use TOTP if specified.
+if [ -n "$TOTP_ISSUER" ]; then
```

Comment:
Is this the best way to do this?  The `totp-issuer` has a default value, per the manual, so should setting the issuer be what triggers activation of this module?  Or should we go with something like `TOTP_ENABLED`?

Reply:
I did not want to add another specific var to keep it as simple as possible, but I can for sure :+1:
Your new var / varname sounds good to me.
Should I ?

Reply:
Done, as it really seems proper 👍